spec/requests/api/v1/accounts/statuses_spec.rb

```diff
@@ -127,9 +127,13 @@
         end
 
         it 'lists both the public and the private statuses' do
-          get :index, params: { account_id: account.id, pinned: true }
-          json = body_as_json
-          expect(json.map { |item| item[:id].to_i }).to contain_exactly(status.id, private_status.id)
+          get "/api/v1/accounts/#{account.id}/statuses", params: { pinned: true }, headers: headers
+
+          expect(body_as_json)
+            .to contain_exactly(
+              include(id: status.id.to_s),
+              include(id: private_status.id.to_s)
+            )
```

Comment:
Same here, for readability, I think it makes more sense this way:
```suggestion
            .to contain_exactly(
              a_hash_including(id: status.id.to_s),
              a_hash_including(id: private_status.id.to_s)
            )
```

Reply:
Updated those instances.

Reply:
Have you pushed the changes? :D

Reply:
I have ... but they are not here yet. Hmm. Maybe https://www.githubstatus.com/incidents/wcl1sw4mzg60

Reply:
Yeah, I just noticed something similar on another PR. Will keep an eye on this.